### PR TITLE
Update cAdvisor to v0.37.0

### PR DESCRIPTION
**Pull request by dashpole**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
cAdvisor does a release for each kubernetes release.  This is the release for k8s 1.19.  It has a number of bug-fixes and improvements.

Because we switched to a different cAdvisor interface for serving prometheus metrics, I had to change a bunch of interfaces in the kubelet.  But the behavior is exactly the same.

**Does this PR introduce a user-facing change?**:
```release-note
Fix detection of image filesystem, disk metrics for devicemapper, detection of OOM Kills on 5.0+ linux kernels.
```

### CODE REVIEW

File: pkg/kubelet/cadvisor/cadvisor_windows.go [hunk 1]
```diff
@@ -58,6 +58,10 @@ func (cu *cadvisorClient) ContainerInfoV2(name string, options cadvisorapiv2.Req
 	return cu.winStatsClient.WinContainerInfos()
 }
 
+func (cu *cadvisorClient) GetRequestedContainersInfo(containerName string, options cadvisorapiv2.RequestOptions) (map[string]*cadvisorapi.ContainerInfo, error) {
+	return nil, nil
```
Comment (cheftako) [inline pkg/kubelet/cadvisor/cadvisor_windows.go:62]: Why is this also not errUnsupported?

Comment (dashpole) [reply pkg/kubelet/cadvisor/cadvisor_windows.go:62]: This PR switches from using SubcontainerInfo to GetRequestedContainersInfo.  The only difference between the two is that SubcontainerInfo takes a request, and GetRequestedContainersInfo takes options.  This is maintaining the same behavior as SubcontainerInfo, which currently returns `nil, nil` as well.

Comment (cheftako) [reply pkg/kubelet/cadvisor/cadvisor_windows.go:62]: Thanks for explaining :+1:

**Comment by dashpole** [comment]
can I get 1.19 milestone as well?

**Comment by dashpole** [comment]
@dims can you help with the milestone?

**Comment by liggitt** [comment]
This is already in the milestone

**Comment by dims** [comment]
cross my fingers 🤞 !!

**Comment by dashpole** [comment]
ah, my bad

**Pull request merged**